TaylorFramework/Modules/finder/Parameters.swift

```diff
@@ -20,21 +20,21 @@ struct Parameters {
     }
     
     init?(dictionary: Options, printer: ErrorPrinter) {
-        let pathValue = dictionary[ParametersKeys.Path]
+        let pathValue = dictionary[ParametersKeys.Path.rawValue]
         if pathValue == nil || pathValue!.isEmpty || pathValue!.first!.isEmpty {
             printer.printWrongRootPathMessage()
             return nil
         }
-        let fileType = dictionary[ParametersKeys.FileType]
+        let fileType = dictionary[ParametersKeys.FileType.rawValue]
         if  fileType == nil || fileType!.isEmpty || fileType!.first!.isEmpty {
             printer.printWrongTypeFile()
             return nil
         }
-        var excludesValue = dictionary[ParametersKeys.Excludes]
+        var excludesValue = dictionary[ParametersKeys.Excludes.rawValue]
         if excludesValue == nil {
             excludesValue = []
         }
-        var filesValue = dictionary[ParametersKeys.Files]
+        var filesValue = dictionary[ParametersKeys.Files.rawValue]
         if filesValue == nil {
             filesValue = []
         }
```

Comment:
Refactor this to `let filesValue = dictionary[ParameterKeys.Files.rawValue] || []`